### PR TITLE
Convert Tpv2 "Users" page to an AG-Grid table

### PR DESCRIPTION
**Pull request by ocket8888**
Switches TPv2's "Users" page to use ag-grid. The old, business-card-esque layout provided little to no actual value, and was significantly less performant than an AG-Grid.

I also had to update Fontawesome to patch a bug.

<hr/>

## Which Traffic Control components are affected by this PR?
None.

## What is the best way to verify this PR?
Make sure the tests pass, start TPv2 and check out the users table.

## If this is a bugfix, which Traffic Control versions contained the bug?
- master

## PR submission checklist
- [x] This PR has tests
- [ ] This PR has documentation
- [ ] This PR has a CHANGELOG.md entry
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY**

### CODE REVIEW

File: experimental/traffic-portal/src/app/shared/table-components/email-cell-renderer/email-cell-renderer.component.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Component } from "@angular/core";
```
Comment (shamrickus) [inline experimental/traffic-portal/src/app/shared/table-components/email-cell-renderer/email-cell-renderer.component.ts:1]: License

File: experimental/traffic-portal/src/app/shared/table-components/telephone-cell-renderer/telephone-cell-renderer.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { Component } from "@angular/core";
```
Comment (shamrickus) [inline experimental/traffic-portal/src/app/shared/table-components/telephone-cell-renderer/telephone-cell-renderer.component.ts:1]: License

**Review by shamrickus — CHANGES_REQUESTED** [review]
Looks good just an issue or two

File: experimental/traffic-portal/src/app/core/core.module.ts [hunk 3]
```diff
@@ -72,6 +73,7 @@ const routes: Routes = [
 	exports: [
 	],
 	imports: [
+		FontAwesomeModule,
```
[the remark this answers is not in the document]
Comment (ocket8888) [reply experimental/traffic-portal/src/app/core/core.module.ts:76]: Oh, I did that when I was trying to figure out why Fontawesome was breaking compilation. Should've known, though - that'd be a runtime error.

**Review by shamrickus — APPROVED** [review]
LGTM

**Pull request merged**